build.sbt

```diff
@@ -2,19 +2,55 @@ inThisBuild(
   List(
     scalaVersion := "2.12.3",
     organization := "org.scalameta",
-    version := "0.1-SNAPSHOT",
-    sources.in(Compile, doc) := Nil // faster publishLocal.
+    sources.in(Compile, doc) := Nil, // faster publishLocal
+    licenses := Seq(
+      "Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")
+    ),
+    homepage := Some(url("https://github.com/scalameta/language-server")),
+    developers := List(
+      Developer(
+        "olafurpg",
+        "Ólafur Páll Geirsson",
+        "olafurpg@gmail.com",
+        url("https://geirsson.com")
+      ),
+      Developer(
+        "gabro",
+        "Gabriele Petronella",
+        "gabriele@buildo.io",
+        url("https://github.com/gabro")
+      ),
+      Developer(
+        "laughedelic",
+        "Alexey Alekhin",
+        "laughedelic@gmail.com",
+        url("https://github.com/laughedelic")
+      )
+    ),
+    scmInfo in ThisBuild := Some(
```

Comment:
This is not necessary, I think. Automatically set by the plugin.

Reply:
Last time I tried it was necessary, but probably you fixed it later. I'll try to remove it.

Reply:
Apparently, it's still needed: https://travis-ci.org/scalameta/language-server/jobs/306019885#L2510